### DISCUSSION TITLE
Roberts Lab Handbook is live!

### DISCUSSION THREAD

**Discussion by kubu4** [RobertsLab/resources#1078] [Show and tell]
Here it is!

https://robertslab.github.io/resources/

I'll continue to work on this to incorporate our Mox and code wikis into this document.

Important to note (and I'll add this note to the respective wikis) that the wikis should be considered "deprecated". This means that the wiki markdown files should no longer be edited, as their content is not "synced" to the handbook. This means that if you want to update the info found in the Roberts Lab Handbook, you will need to edit the various markdown files here:

https://github.com/RobertsLab/resources/tree/master/docs

Please discuss with thoughts/critiques/complements/etc.

This documentation was built using [MkDocs](https://www.mkdocs.org/), following the [readthedocs](https://docs.readthedocs.io/en/stable/) guidelines.

**Comment by sr320** [comment]
Beautiful!
Neat that each doc has TOC on right.

I presume there is some way to customize the left sidebar (group by theme as opposed to alphabetical)?

**Reply by sr320** [reply]
Possibly as simple as editing https://github.com/RobertsLab/resources/blob/master/mkdocs.yml?

**Reply by sr320** [reply]
For later reference - this might be useful https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/

**Reply by kubu4** [reply]
> Possibly as simple as editing https://github.com/RobertsLab/resources/blob/master/mkdocs.yml?

Yep, the list in the `nav` section governs the list order of the left sidebar. I simply listed them in alphabetical order since I made the list programmatically.
